charts/rollout-app/templates/services.yaml

```diff
@@ -60,13 +59,11 @@ spec:
 This optional "Canary" service is created only when the .Values.strategy is set
 to "canary". Rollouts will then use this service to route canary traffic to, in
 order to test releases.
-
 When using Istio VirtualServices (if .Values.virtualService.enabled=true), the
```

Comment:
Is there any case where if someone wanted to have 1 rollout per AZ, that they'd also want 1 virtual service per AZ? I think the "umbrella" service concept would be needed as well so that the region's service as a whole can have a single ingress URL for clients to access it from other services. Otherwise the client has to know which AZ they're in and change the host in the URL for that AZ, which we don't do currently.